test/test_config.py

```diff
@@ -21,7 +24,51 @@ def get_test_config():
         },
         "datasets": [{"label": "alg1"}, {"label": "alg2"}],
         "gold_standards": [{"label": "gs1", "dataset_labels": []}],
-        "algorithms": [{"params": ["param2", "param2"]}],
+        "algorithms": [
+            {"params": ["param2", "param2"]},
+            {
+                "name": "strings",
+                "params": {
+                    "include": True,
+                    "run1": {"test": "str1", "test2": ["str2", "str3"]}
```

Comment:
I must be misreading something. I thought I single string was not currently allowed, but it works in the test here.

Reply:
The `obj = [obj]` is a catch-all for non-evaluated objects - it handles strings. I've just documented that behavior :+1: